pyproject.toml

```diff
@@ -33,7 +33,7 @@ keywords = [
 [tool.poetry.dependencies]
 python = "^3.8"
 python-telegram-bot = "^13.3"
-django = "2.2.19"
+django = "2.2.20"
```

Comment:
Please use django = "~2.2"
otherwise, it can be a nightmare with adopting projects.

Reply:
fixed